src/api/users.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const getUsers = () => {
```

Comment:
Consider adding error handling to the `getUsers` function to manage potential API request failures gracefully. This will help in providing feedback to the user in case of network issues or server errors.